vic/drivers/cesm/bld/vic.buildnml.csh

```diff
@@ -0,0 +1,3 @@
+#! /bin/csh -f
+
+cd $RUNDIR
```

Comment:
@apcraig - 

I think this is going to be my next task.  Can you walk me through how you think we should configure the namelist build step?  VIC has one "namelist" that we call a global configuration file.  It is just a simple ASCII file that looks something like this:

```
# VIC Model Parameters for CESM driver

# Simulation Parameters
SNOW_STEPS_PER_DAY	72
RUNOFF_STEPS_PER_DAY  72

FULL_ENERGY TRUE
FROZEN_SOIL	TRUE

# Simulation Options
AERO_RESIST_CANSNOW AR_406

# DOMAIN INFO
DOMAIN ${DIN_LOC_ROOT}lnd/vic/griddata/fracdata_wr50a_ar9v4_101013.nc

FULL_ENERGY TRUE
FROZEN_SOIL	FALSE
```

So we'll have a default global config file that needs to have the variables like `${DIN_LOC_ROOT}l` expanded. I can take care of all that in `vic.buildnml.csh`.  There is also `user_nl_vic` in the `${CASE}` directory.  What is the CESM standard on how to use that? Ideally, we won't need to edit `vic.buildnml.csh` to change the VIC configuration, right?